### ISSUE TITLE
Declaration files not emitted when declarationDir option is specified

### ISSUE THREAD

**Issue by red-0ne** [s-panferov/awesome-typescript-loader#247]
Declaration files are not emitted if the `declarationDir` option is present in the `tsconfig.json` file.cat

webpack.config.js

``` javascript
var webpackCfg = {
    entry: {
        main: "./src/init-client.ts"
    },
    output: {
        filename: 'dist/[name].js'
    },
    resolve: {
        extensions: ['.ts', '.js']
    },
    module: {
        loaders: [
            { test: /\.ts$/, loader: 'awesome-typescript-loader' }
        ]
    }
};

module.exports = webpackCfg;
```

tsconfig.json

``` json
{
    "compilerOptions": {
        "target": "es5",
        "module": "commonjs",
        "declaration": true,
        "experimentalDecorators": true,
        "emitDecoratorMetadata": true,
        "declarationDir": "decl/",
        "rootDir": "src/",
        "outDir": "dist/"
    },
    "files": [
        "src/init-client.ts"
    ]
}
```

`decl` directory should be populated

**Comment by Lovelidge** [comment]
It looks like a bug. I was running into the same problem. It's placing the decl/ folder as a sibling of your project folder (<project folder>). As a work around, I would suggest the following: 

"compilerOptions": {
    "declarationDir": "<project folder>/decl",
}

**Comment by s-panferov** [comment]
Thanks, landed into awesome-typescript-loader@3.1.0
